oh-fitbit-integration/settings.py

```diff
@@ -58,6 +60,10 @@
     FITBIT_CALLBACK_URL =\
         'http://oh-fitbit-integration.herokuapp.com/complete/fitbit'
 
+# This creates a Realm called "source" that allows 60 requests per minute max.
```

Comment:
I think we want to add the `if REMOTE is True` stuff from the demo template, that sets up Redis with a password for production deploys. See https://github.com/OpenHumans/oh-data-demo-template/blob/master/demotemplate/settings.py#L55

Reply:
ok I did that